packages/story-editor/src/typings/global.d.ts

```diff
@@ -13,10 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+export {};
 
-/**
- * External dependencies
- */
-import { createContext } from '@googleforcreators/react';
-
-export default createContext({ state: {}, actions: {} });
+declare global {
+  interface Array<T> {
+    findLastIndex(
+      predicate: (value: T, index: number, obj: T[]) => unknown,
+      thisArg?: any
+    ): number;
+  }
```

Comment:
This smells like we refer to the wrong JS version somewhere. @swissspidy?

Reply:
Nope. `findLastIndex` is a stage 3 proposal. See https://chromestatus.com/feature/5693639729610752 and https://github.com/microsoft/TypeScript/issues/48829#issuecomment-1147769106

Reply:
Oh wow, I thought it had been around for ages. Well, go to know.